compiler/x/amd64/codegen/OMRTreeEvaluator.cpp

```diff
@@ -2198,7 +2198,14 @@ OMR::X86::AMD64::TreeEvaluator::inotzEvaluator(TR::Node *node, TR::CodeGenerator
 TR::Register*
 OMR::X86::AMD64::TreeEvaluator::ipopcntEvaluator(TR::Node *node, TR::CodeGenerator *cg)
    {
-   return TR::TreeEvaluator::badILOpEvaluator(node, cg);
+   TR::Node *child = node->getFirstChild();
+   TR::Register *inputReg = cg->intClobberEvaluate(child);
+
+   generateRegRegInstruction(TR::InstOpCode::POPCNT4RegReg, node, inputReg, inputReg, cg);
+
+   node->setRegister(inputReg);
+   cg->decReferenceCount(child);
+   return inputReg;
```

Comment:
As the implementations of `OMR::X86::AMD64::TreeEvaluator::ipopcntEvaluator` and `OMR::X86::I386::TreeEvaluator::ipopcntEvaluator` are identical, is there a reason they shouldn't be unified as `OMR::X86::TreeEvaluator::ipopcntEvaluator` and moved into [`compiler/x/codegen/UnaryEvaluator.cpp`](https://github.com/eclipse-omr/omr/blob/f92a3f02892bb78c4eb7d12a2de4040ffb31aa28/compiler/x/codegen/UnaryEvaluator.cpp)?